plugin/trino-exasol/src/test/java/io/trino/plugin/exasol/TestExasolConnectorTest.java

```diff
@@ -327,6 +350,20 @@ void testNativeMultipleInClauses()
         onRemoteDatabase().execute(format("SELECT count(*) FROM %s.orders WHERE %s", TEST_SCHEMA, longInClauses));
     }
 
+    @Override
+    @Test
+    public void testNativeQueryInsertStatementTableExists()
+    {
+        abort("Read-only Exasol connector does not support CREATE TABLE and INSERT statements");
```

Comment:
Please verify the behavior instead of `abort`. Same for `testNativeQuerySelectFromTestTable`.